### ISSUE TITLE
Use Size Limit

### ISSUE THREAD

**Issue by ai** [irustm/ng-storeon#2]
Using [Size Limit](https://github.com/ai/size-limit/) instead of Bundlephobia can bring some benefits:

1. All other Storeon tools use it.
2. You can get bundlephobia numbers only after release. Size Limit can be used during development.
3. Size Limit will prevent mistakes: is somebody will send PR, Size Limit will check it on CI.
4. Bundlephobia calculation is not 100% accurate. Bundlephobia number contains webpack boilerplate, but it is not part of the cost since the user already has it.
5. Last but not least, Size Limit tracks time to compile your JS.

**Comment by irustm** [comment]
done,
https://travis-ci.org/irustm/ng-storeon
